README-gradle.md

```diff
@@ -0,0 +1,97 @@
+## Using iDFlakies on a Gradle project
+
+### Install iDFlakies and testrunner 
+
+Go to the testrunner root directory, and run: 
+
+```
+mvn clean install
+```
+
+Then go to iDFlakies root directory, and run: 
+
+```
+mvn clean install
+```
+
+### Automatically setting up the build.gradle for iDFlakies
+
+Run the following command to  automatically setup the build.gradle for iDFlakies.
+
+```
+bash gradle_modify/modify_gradle.sh path_to_gradle_project
+```
+
+### Manually setting up the build.gradle for iDFlakies
+
+We can configure the build.gradle file manually instead. We need to follow the rule listed below to successfully set up the project. 
+
+1. First we need to add the following dependencies and repository to the `buildscript{}` in build.gradle
+
+   ```groovy
+   buildscript {
+      repositories {
+         ...
+         mavenLocal()
+         ...
+      }
+   
+      dependencies {
+         ...
+         classpath group: 'edu.illinois.cs',
+         name: 'idflakies', 
+         version: '1.2.0'
```

Comment:
I have deployed iDFlakies as version 1.1.0 instead (the version should be live in about one day). Please update this document globally